recipe/meta.yaml

```
@@ -78,40 +78,42 @@ test:
 {% endif %}
 {% if cuda_major == 12 %}
     - cuda-cudart-dev
-    - cuda-driver-dev
+    - cuda-driver-dev   # [not win]
 {% endif %}
   commands:
   # Lots more CMake files; just checking for one
     - test -f $PREFIX/lib/cmake/Kokkos/KokkosConfig.cmake  # [unix]
-    - if not exist %LIBRARY_LIB%\\cmake\\Kokkos\\KokkosConfig.cmake  # [win]
+    - if not exist %LIBRARY_LIB%\\cmake\\Kokkos\\KokkosConfig.cmake exit 1  # [win]
   # Testing explicity for all shared libs, but not links
     - test -f $PREFIX/lib/libkokkoscontainers.so.{{ so_version }}  # [linux]
     - test -f $PREFIX/lib/libkokkoscore.so.{{ so_version }}  # [linux]
     - test -f $PREFIX/lib/libkokkossimd.so.{{ so_version }}  # [linux]
     - test -f $PREFIX/lib/libkokkoscontainers.{{ so_version }}.dylib  # [osx]
     - test -f $PREFIX/lib/libkokkoscore.{{ so_version }}.dylib  # [osx]
     - test -f $PREFIX/lib/libkokkossimd.{{ so_version }}.dylib  # [osx]
-    - if not exist %LIBRARY_LIB%\\libkokkoscontainers.lib exit 1  # [win]
-    - if not exist %LIBRARY_LIB%\\libkokkoscore.lib exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\\kokkoscontainers.lib exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\\kokkoscore.lib exit 1  # [win]
```

```suggestion
    - if not exist %LIBRARY_LIB%\\kokkoscontainers.lib exit 1  # [win]
    - if not exist %LIBRARY_LIB%\\kokkoscore.lib exit 1  # [win]
    - if not exist %LIBRARY_LIB%\\kokkossimd.lib exit 1  # [win]
    - if not exist %LIBRARY_BIN%\\kokkoscontainers.dll exit 1  # [win]
    - if not exist %LIBRARY_BIN%\\kokkoscore.dll exit 1  # [win]
    - if not exist %LIBRARY_BIN%\\kokkossimd.dll exit 1  # [win]
```
Add something like this.

---

done